nbs/core.ipynb

```diff
@@ -192,11 +192,19 @@
     "        and datestamps with the `unique_id` and `ds` columns. The `y` column denotes the target \n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #44.                df: pd.DataFrame,
</pre><p>The df is optional because it can also be loaded from a stored checkpoint (see assertion in line 135 and load/save methods).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/378/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
thanks! fixed in [8390f36](https://github.com/Nixtla/neuralforecast/pull/378/commits/8390f36dc3bb203ec6ed671150df86fdcc77f26b)